compiler/rustc_error_codes/src/error_codes.rs

```diff
@@ -417,7 +417,6 @@ E0687: include_str!("./error_codes/E0687.md"),
 E0688: include_str!("./error_codes/E0688.md"),
 E0689: include_str!("./error_codes/E0689.md"),
 E0690: include_str!("./error_codes/E0690.md"),
-E0691: include_str!("./error_codes/E0691.md"),
```

Comment:
I have removed the code here but kept the E0691.md file. I would thing that's wrong because the file is now dangling, but further down in this file there is a claim saying that "many removed error codes are documented" so maybe this is the intention? This file lacks guidance for what to do when an error code is being removed. Someone should please decide that and document it.

Reply:
Oh, and then *after all that* now CI tells me to never remove error check conditions. That's really frustrating. At this point I spent several roundtrips and >10min just on figuring just how to remove an error code, and I am getting conflicting advice from tidy, the in-file comments, and validate-error-codes.sh.

Reply:
https://github.com/rust-lang/rust/pull/115278